iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -97,7 +97,15 @@ final class MainViewController: UIViewController, CardViewControllerDelegate, Th
 
         let paymentConfiguration = PaymentFormConfiguration(apiKey: "pk_test_6e40a700-d563-43cd-89d0-f9bb17d35e73",
                                                             environment: environment,
-                                                            supportedSchemes: [.visa, .mastercard, .maestro],
+                                                            supportedSchemes: [.visa, .visa, .visa,
```

Comment:
multi .visa is added like to this to show uniqueness of supported scheme

Reply:
Does this include visa just once then ?

Reply:
yes, unique schemes